cmp/filter/FilterChooserFieldSpec.js

```diff
@@ -239,23 +158,14 @@ export class FilterChooserFieldSpec extends HoistBase {
         return 'value';
     }
 
-    parseOperators(ops) {
-        ops = ops ?? this.getDefaultOperators();
-        return ops.filter(it => FieldFilter.OPERATORS.includes(it));
-    }
-
-    getDefaultOperators() {
-        if (this.isBoolFieldType) return ['='];
-        return this.isValueType ? ['=', '!=', 'like'] : ['>', '>=', '<', '<=', '=', '!='];
-    }
-
-    loadValuesFromStore() {
-        const {field, store} = this,
+    loadValuesFromSource() {
+        const {field, source} = this,
             values = new Set();
 
         // Note use of unfiltered recordset here to source suggest values. This allows chooser to
```

Comment:
interesting that this behavior is exactly opposite what we are doing with the column filters enumeration